sample.env

```diff
@@ -48,3 +48,13 @@ WATCH_EPICGAMES=false
 
 # Comma-separated Applications IDs to track.
 EPICGAMES_APP_IDS="963137e4c29d4c79a81323b8fab03a40, bcbc03d8812a44c18f41cf7d5f849265, Kinglet"
+
+# GOG ------------------------------------------------
+# Set "true" if you want to track the products on GOG.
+WATCH_GOG=false
+
+# Comma-separated Product IDs to track on GOG.
+# You can also specify branch name by "<Product ID>:<Branch Name>".
+# If no branch name is specified, defaults to null branch.
+GOG_APP_IDS="1193046833, 1193046833:experimental - latest unstable version"

```

Comment:
Could you please update `README.md` for users to describe how to find these IDs and branches.
I don't have account on GOG so I have no idea how to achieve that.

Reply:
The API I used to find IDs is` https://embed.gog.com/games/ajax/filtered?search=[game name]` which gives a massive blob of JSON as output
You can also find IDs by looking in the HTML code of https://www.gog.com/game/[game name]
Neither of those are nice solutions for end users, so I'm thinking of modifying `app_finder.py` (which I now realize needs updates for GOG anyway) to search for names - do you think that's a good solution?

Reply:
Thanks for your confirmation. Agree, fixing `app_finder.py` is the most ideal solution.

That said, the number of users of this tool is probably very small, so honesty I think we do not need to spend too much time on development😅

If you enjoy developing it, it would be super helpful that you implement this, but if it is too much of a burden for you, just add a brief description to the `README.md` is enough.

Reply:
Done